components/resources/BUILD.gn

```diff
@@ -60,19 +60,21 @@ repack("resources") {
   deps = [ ":static_resources" ]
   sources = [ "$root_gen_dir/components/brave_components_static.pak" ]
 
+  deps += [ "//brave/components/skus/browser/resources:generated_resources" ]
+
+  sources += [ "$root_gen_dir/brave/components/skus/browser/resources/skus_internals_generated.pak" ]
+
```

Comment:
These can just be added to the `deps`/`sources` declarations above, no need to append